service/src/main/java/com/ge/predix/acs/encryption/Encryptor.java

```diff
@@ -0,0 +1,94 @@
+package com.ge.predix.acs.encryption;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.util.Arrays;
+
+import javax.crypto.Cipher;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import org.apache.commons.codec.binary.Base64;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+
+import com.google.common.primitives.Bytes;
+
+// CHECKSTYLE:OFF: FinalClass
+public class Encryptor {
+
+    @Value("${ENCRYPTION_KEY}")
+    private String encryptionKey;
+
+    private static final Encryptor INSTANCE = new Encryptor();
+    private static final Logger LOGGER = LoggerFactory.getLogger(Encryptor.class);
+
+    private static final String ALGO_WITH_PADDING = "AES/CBC/PKCS5PADDING";
+    private static final String ALGO = "AES";
+    private static final String ENCODING = "UTF-8";
+    private static final int IV_LENGTH_IN_BYTES = 16;
+    private static final int KEY_LENGTH_IN_BYTES = 16;
+
+    private Cipher cipher;
+
+    private Encryptor() {
+        try {
+            this.cipher = Cipher.getInstance(ALGO_WITH_PADDING);
+        } catch (NoSuchAlgorithmException | NoSuchPaddingException e) {
+            LOGGER.error("Can not created instance of cipher with algorithm" + ALGO_WITH_PADDING);
+        }
+    }
+
+    public static Encryptor getInstance() {
+        return INSTANCE;
+    }
+
+    public String encrypt(final String value) {
+        validateKeyLength();
+        try {
+            byte[] ivBytes = new byte[IV_LENGTH_IN_BYTES];
+            SecureRandom.getInstanceStrong().nextBytes(ivBytes);
+            IvParameterSpec iv = new IvParameterSpec(ivBytes);
+            SecretKeySpec skeySpec = new SecretKeySpec(this.encryptionKey.getBytes(ENCODING), ALGO);
+
+            this.cipher.init(Cipher.ENCRYPT_MODE, skeySpec, iv);
+
+            byte[] encrypted = this.cipher.doFinal(value.getBytes());
+            byte[] result = Bytes.concat(ivBytes, encrypted);
```

Comment:
IV is required by CBC.  We could switch to using ECB instead and then IV will not be required.  ECB is considered weaker than CBC since the same blocks of plain text encrypted with ECB result in the same encrypted blocks.